app/src/main/java/com/owncloud/android/ui/activity/SettingsActivity.java

```diff
@@ -234,83 +235,72 @@ private void setupDevCategory(PreferenceScreen preferenceScreen) {
         }
     }
 
-    private void setupAboutCategory(String appVersion) {
-        final PreferenceCategory preferenceCategoryAbout = (PreferenceCategory) findPreference("about");
+    private void setupDataPrivacyCategory() {
```

Comment:
For maintenance reason, please do not change such a method, but instead create a new method and just disable "about category".